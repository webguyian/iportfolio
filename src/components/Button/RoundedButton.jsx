import React from 'react';
import PropTypes from 'prop-types';
import classNames from 'classnames';

import Button from 'components/Button/Button';

const RoundedButton = props => {
  const baseClass = 'ui-btn--rounded';

  return (
    <Button {...props} className={classNames(baseClass, props.className)} />
  );
};

RoundedButton.propTypes = {
  className: PropTypes.string
};

export default RoundedButton;
