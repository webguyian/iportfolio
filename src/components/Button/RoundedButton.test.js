import React from 'react';
import renderer from 'react-test-renderer';

import RoundedButton from './RoundedButton';

describe('<RoundedButton />', () => {
  it('renders correctly', () => {
    const component = renderer.create(<RoundedButton />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
