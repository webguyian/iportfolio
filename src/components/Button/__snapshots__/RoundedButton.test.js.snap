// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RoundedButton /> renders correctly 1`] = `
<button
  className="ui-btn ui-btn--rounded"
  type="button"
/>
`;
